### ISSUE TITLE
[User story] ....

### ISSUE THREAD

**Issue by pazbardanl** [Green-Software-Foundation/if-models#25]
As a user of IF `MockObservations `plugin I want to be able to define the time buckets of my mocked observations using the start time and number of samples so that I don't necessarily need to provide the end time if it is insignificant to my usage.

**Comment by jmcook1186** [comment]
@pazbardanl is this issue still relevant?

**Comment by jmcook1186** [comment]
@pazbardanl is this issue still relevant?

**Comment by pazbardanl** [comment]
Yes. It's not a major change to make. Not sure what's the priority of improving MockObservations though. Your call I guess :) @jmcook1186
